Api/v2 (#8)
ministryofjustice/fb-base-adapter: README.md

# Form Builder Base Adapter

This is used for simulating the potential JSON endpoint adapters that integrate with the Form Builder platform.

## Mechanism

> The base adapter receives submissions adn store temporarily into a file.

Reviewer: ```suggestion
The base adapter receives submissions and stores it temporarily as a file.
```
